sdk/cosmos/azure-cosmos-spark_3-0_2-12/Samples/Pyspark Sample.ipynb

```diff
@@ -0,0 +1,159 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "inputWidgets": {},
+     "nuid": "8d122d67-03ea-4684-af8d-7c68e4d6bc77",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# configuration\n",
+    "cosmosEndpoint = \"https://REPLACEME.documents.azure.com:443/\"\n",
```

Comment:
Non-Blocking - Need to follow-up with Revin to check whether there is any reasonably easy way to use AAD auth here. Otherwise the official samples later should probably use Scala secrets by default (it is a trade-off thought - this one is simpler - on the other hand it is usually a good practice to apply security best practices in the samples)